tools/update-locales.js

```diff
@@ -0,0 +1,77 @@
+#!/usr/bin/env node
+'use strict';
+
+const fs = require('fs-extra');
+const path = require('path');
+const unzip = require('unzip');
+
+const root = path.join(__dirname, '..');
+const localeFolder = path.join(root, '_locales');
+const tempFolder = path.join(root, '_temp');
+const zipFile = path.join(root, 'github-7_Stylus_messages.zip');
+
+/*
+ * Download transifex zip file into the root folder before running this script
```

Comment:
Should we move the downloading process into this script?

Reply:
Hmm, you're right... using the Transifex client is easier. I'll update this PR.